skopeo/README.md

```diff
@@ -38,3 +38,24 @@ podman run docker://quay.io/skopeo/stable:latest copy docker://quay.io/skopeo/st
 # Inspect the fedora:latest image
 podman run docker://quay.io/skopeo/stable:latest inspect --config docker://registry.fedoraproject.org/fedora:latest  | jq
 ```
+
+## Use against a private registry server is possible but somewhat complex
+
+1. Assuming one isn't already defined, setup a podman secret with the `auth.json` contents.
+   See the `containers-auth.json` man page for the format.
+
+```
+$ REGISTRY_AUTH_FILE=$(mktemp)
+$ podman login example.com/registry
+$ podman secret create registry_name_auth "$REGISTRY_AUTH_FILE"
+$ rm "$REGISTRY_AUTH_FILE"
```

Comment:
Run all of this in a subshell so that `REGISTRY_AUTH_FILE` does not affect subsequent operations.

Better, use `podman login --authfile …` and some other private per-script variable to capture the `mktemp` output:
```console
$ myTmpFile=$(mktemp)
$ podman login --authfile ${myTmpFile} …
```

Reply:
Oof, this was my original approach.  Then I thought this might highlight `REGISTRY_AUTH_FILE` as being magic.  But no, you're right, this way is dangerous.